src/components/site-header/_site-header-dropdown.scss

```diff
@@ -335,3 +334,10 @@ $language-list: null !default;
     width: calc(#{map.get($language-list, 'item-width')} - (2 * var(--s-s)));
   }
 }
+
+@include breakpoints.up('l') {
+  .ecl-site-header__custom-action-overlay {
+    right: 50%;
```

Comment:
this will not work if the custom action is on the right of the screen (no search or language switcher)
![image](https://github.com/user-attachments/assets/ea7da904-9e4d-40c6-96b3-f897b334bf42)
I don't really understand: why do we have a custom behavior here, instead of reusing what we have for the language switcher?
The language switcher overlay has some custom content for sure, but the way it is placed on the screen is generic, if I'm not wrong

Reply:
i did not want to use it because it needed to be refactored because it was hardcoded for language list. In any case, i refactored and now supports also custom action. Also fixed some issues on eu.